config/routes.rb

```diff
@@ -0,0 +1,13 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root 'static_pages#home'
+  get '/signup', to: 'users#new'
+  post '/signup', to: 'users#create'
+  get '/login', to: 'sessions#new'
+  post '/login', to: 'sessions#create'
+  delete '/logout', to: 'sessions#logout'
+  get '/posts', to: 'posts#new'
+  post '/posts', to: 'posts#create'
+  get '/index', to: 'posts#index'
+  resources :users
```

Comment:
- [ ] we should be creating the routes with resources in all the cases, requirement [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_routes_requirement.png)